File: app/src/main/java/me/cpele/traintimer/MainActivity.java
package me.cpele.traintimer;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        RecyclerView recyclerView = findViewById(R.id.main_rv);
        TimerAdapter adapter = new TimerAdapter();
        adapter.add(new TimerItem(
                1540552080,
                "Gare de Lyon",
                1540561680,
                "Dax"
        ));
        adapter.add(new TimerItem(
                1540552080,
                "Gare d'Austerlitz",
                1540561680,
                "Montbard"
        ));
        recyclerView.setAdapter(adapter);
        recyclerView.setLayoutManager(new LinearLayoutManager(this));
    }
}

File: app/src/main/java/me/cpele/traintimer/MainActivity.kt
package me.cpele.traintimer

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import java.util.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val recyclerView = findViewById<RecyclerView>(R.id.main_rv)
        val adapter = TimerAdapter()
        adapter.add(TimerItem(
                GregorianCalendar(2018, Calendar.FEBRUARY, 5, 13, 42, 23).time,
                "Gare de Lyon",
                GregorianCalendar(2018, Calendar.FEBRUARY, 5, 17, 41, 59).time,
                "Dax"
        ))
        adapter.add(TimerItem(
                GregorianCalendar(2018, Calendar.APRIL, 23, 12, 24, 16).time,
                "Gare d'Austerlitz",
                GregorianCalendar(2018, Calendar.APRIL, 23, 15, 22, 19).time,
                "Montbard"
        ))
        recyclerView.adapter = adapter
        recyclerView.layoutManager = LinearLayoutManager(this)
    }
}
